src/session9/challenges/college_management_system/Professor.java

```diff
@@ -0,0 +1,81 @@
+package session9.challenges.college_management_system;
+
+public class Professor {
+
+    private String firstName;
+    private String lastName;
+    private String sex;
+    private int age;
+    private String dateOfBirth;
+    private String CNP;
+    private String address;
```

Comment:
We can fit the whole address in a single String? Can we use maybe a custom object for address propertie, where we can have properties like `country`, `city`, `street`, etc?